src/components/view/SurveyTasksView/SurveyTasksView.tsx

```diff
@@ -13,7 +13,11 @@ export interface SurveyTasksViewProps {
 
 export type ViewPresentationType = "Modal" | "Push";
 
-export default function (props: SurveyTasksViewProps) {
+/**
+ * This ready-to-use view displays all tasks.
+ * It can be configured to show or hide completed and incomplete tasks as needed.
```

Comment:
Suggested edit:

`It can be configured to hide completed and incomplete tasks as needed.`